src/exberry_int/exberry_integration.py

```diff
@@ -0,0 +1,250 @@
+import asyncio
+from aiohttp import ClientSession, WSMsgType
+from dataclasses import dataclass
+import hashlib
+import hmac
+import itertools
+from typing import Type
+import time
+from logging import Logger
+from enum import Enum
+from .types import Endpoints
+
+from daml_dit_if.api import \
+    IntegrationEnvironment
+
+@dataclass
+class ExberryIntegrationEnv(IntegrationEnvironment):
+    username: str
+    password: str
+    tradingApiUrl: str
+    adminApiUrl: str
+    apiKey: str
+    secret: str
```

Comment:
Have you thought about putting the credentials and secrets into a contract? (We do that with the Slack integration, and it nicely hides secrets, etc. using ledger facilities.)

Reply:
Yeah, I did see that in the Slack integration. I've been planning to update this integration to use that at well, but it would be in another PR.

Reply:
Be thinking if you want specific support for config contracts in `daml-dit-if` - right now, you just have to subscribe to contract create events.